BootloaderCommonPkg/BootloaderCommonPkg.dec

```diff
@@ -285,3 +285,4 @@
   # Control if X2APIC should be used or not
   gPlatformCommonLibTokenSpaceGuid.PcdCpuX2ApicEnabled            | FALSE  | BOOLEAN | 0x20000220
   gPlatformCommonLibTokenSpaceGuid.PcdTccEnabled                  | FALSE  | BOOLEAN | 0x20000221
+  gPlatformCommonLibTokenSpaceGuid.PcdSimicsEnabled               | FALSE  | BOOLEAN | 0x20000222

```

Comment:
I think we should avoid using Simics PCD.
Can we do runtime board detection and then have fixed board ID for Simics ?

Reply:
Done.